patches/minecraft/net/minecraft/item/Item.java.patch

```diff
@@ -352,17 +352,21 @@
 +    }
 +
 +    /**
-+     * <em><strong>MODDERS</strong>: Override this to set a non-default {@link EntityEquipmentSlot}
-+     * for an itemstack, but <strong>do not use this to get the {@link EntityEquipmentSlot} for an
++     * Override this to set a non-default armor slot
++     * for an itemstack, but <em>do not use this to get the armor slot of an
```

Comment:
Call it a "stack" or call it `ItemStack` (as code), but don't leave it as "itemstack."